src/ducks/transactions/TransactionSelectDates.jsx

```diff
@@ -22,15 +28,22 @@ export const getOptions = transactions => {
   const mAvailableMonths = new Set(availableMonths)
 
   const start = parse(availableMonths[0], 'YYYY-MM')
-  const end = new Date()
+  const lastMonth = parse(
+    availableMonths[availableMonths.length - 1],
+    'YYYY-MM'
+  )
+  const today = new Date()
+  const end = isAfter(lastMonth, today) ? lastMonth : today
 
-  return rangeMonth(start, end).map(month => {
+  const result = rangeMonth(start, end).map(month => {
```

Comment:
you can return directly this value

Reply:
oh yeah that was just a test that I forgot to revert. Thanks